.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos: 
```

Comment:
pre-commit's configuration:
- sets the requirement to install prettier
- restricts prettier's domain to JSON